server.ts

```diff
@@ -50,7 +50,10 @@ import { AdminAvailableStudents } from './src/db_queries/admin_available_student
 import { AdminPartnersMentors } from './src/db_queries/admin_partners_mentors';
 import { AdminPartnersMentorStats } from './src/db_queries/admin_partners_mentors_stats';
 import { AdminPartnersProjects } from './src/db_queries/admin_partners_projects'
-import {AdminPartnersStudents} from "./src/db_queries/admin_partners_students";
+import { AdminPartnersStudents } from "./src/db_queries/admin_partners_students";
+import {
+  AdminPartnersOrganizationCentersWithStudents
```

Comment:
Let's use `centre`/`centres` instead of `center`/`centers`, as this is the naming convention that we've used so far.